code/modules/shuttle/emergency.dm

```diff
@@ -126,6 +134,11 @@
 	. = TRUE
 	process()
 
+/obj/machinery/computer/emergency_shuttle/proc/clear_recent_action(mob/user)
+	acted_recently -= user
+	if (!QDELETED(user))
+		ui_interact(user)
```

Comment:
Would this not reopen the UI if they close it while standing next to it?

Reply:
You are correct on that, I was able to reproduce that in game.
After the timeout, the UI would re-open. Should be fine to remove it.

I'm unsure if there needs to be the call to "ui_interact(user)" in the main "ui_act" proc either.

Reply:
Does the UI not auto-update?

Reply:
UI does auto-update without the calls